addon-test-support/index.js

```diff
@@ -7,7 +7,6 @@ export {
   count,
   fillable,
   hasClass,
-  is,
```

Comment:
I think Addepar uses this re-export in the main codebase.

Reply:
I'll double-check, but I don't think it's possible. `import { is } from 'ember-class-page-object';` should give you an `is === undefined`